build/config/PublishData.json

```diff
@@ -50,6 +43,13 @@
             "nuget": [ "https://api.nuget.org/v3/index.json" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.3" ]
+        },
+        "dev15.6": {
+            "nugetKind": "Release",
+            "version": "2.7.*",
```

Comment:
Since GitHub doesn't let me comment on the line directly:

we also need to bump the "version" in master-vs-deps to match this.

Reply:
Any changes to the channel for the master-vs-deps ?

---
In reply to: [160279018](https://github.com/dotnet/roslyn/pull/24109#discussion_r160279018) [](ancestors = 160279018)

Reply:
@VSadov Yes, update it too.